plugins/newsdownloader.koplugin/main.lua

```diff
@@ -47,13 +47,7 @@ function NewsDownloader:addToMainMenu(menu_items)
         if not lfs.attributes(NEWS_DL_DIR, "mode") then
             lfs.mkdir(NEWS_DL_DIR)
         end
-
-        FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
-        if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
-            logger.dbg("NewsDownloader: Creating init configuration")
-            FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
-                             FEED_CONFIG_PATH)
-        end
+	FEED_CONFIG_PATH = NEWS_DL_DIR .. FEED_CONFIG_FILE
```

Comment:
`four spaces :)` I used tab there. Do you mean I should replace that with (doubled) 4 spaces?
It looks fine in Gedit. 

In Javascript there something like ["Minify"](https://en.wikipedia.org/wiki/Minification_(programming)). 
Maybe there also something like that for lua/travis?
As travis is written in ruby I bet there some tools for that.
It would make more sense to use a tool than manual removal tailing whitespaces, changing tabs to 4 spaces etc? (But I guess it's a good thema for new topic)

What do you want me to do with that? replace the tab with spaces?